x-pack/solutions/security/packages/ecs-data-quality-dashboard/impl/data_quality_panel/actions/chat/index.tsx

```diff
@@ -44,21 +45,42 @@ const ChatActionComponent: FC<Props> = ({ indexName, markdownComment, checkedAt
   }, [checkedAt, indexName]);
 
   const styles = useStyles();
-  const { isAssistantEnabled } = useDataQualityContext();
+  const { isAssistantEnabled, httpFetch, toasts } = useDataQualityContext();
+  const {
```

Comment:
I tried for about 45 minutes to add tests for this, but the mocking logic in this package is extremely difficult to follow. Given the approaching feature freeze and lack of time, I ask that @elastic/security-threat-hunting add tests to this PR or forgive the missing tests.